nginx_stage/lib/nginx_stage/configuration.rb

```diff
@@ -512,6 +522,8 @@ def set_default_configuration
       self.user_regex     = '[\w@\.\-]+'
       self.min_uid        = 1000
       self.disabled_shell = '/access/denied'
+      self.disabled_shell_message = 'user has a disabled shell: %s'
+      self.disable_nginx_stage_help_message = false
```

Comment:
I don't like double negatives here. Instead of `disable` set to `false` can we have just `nginx_stage_help_message` set to `true`?

Or maybe it's `show_nginx_stage_help_message` set to `true`?

It's just the double negatives that make it hard to understand at a glance.